### DISCUSSION TITLE
Not routed to Voicemail

### DISCUSSION THREAD

**Discussion by pagmax** [amazon-connect/amazon-connect-salesforce-scv#118] [General]
Requirement is: 
End user dials in the Phone number, end user is auto routed to Voicemail. There will be no agents to receive the call, it will be always Voicemail. End user records a voicemail and disconnects the call. Voicemail recordings are saved in s3 bucket and a salesforce record is created.

a. Installed the AWSSCV-SalesforceConfig template in Amazon Connect. -- All good here.\
b. Tested with SCVSalesforceValidator. -- All good here.
c. Installed the Voicemail Express template and perfrmed all the steps as mentioned.

ISSUE:
Dialling the Phone number, it rings and then says that the call could not be completed at this point of time. Can someone please help in understanding where exactly the issue is? 
@dougjaso Any insights on what might be the issue?

**Comment by dougjaso** [comment]
This would not be a Voicemail Express issue, and would instead be something wrong with the phone number on Connect. None of the Voicemail flows have anything particular in them that could cause the flow to execute at all. Have you tried a different flow with the number?
